src/components/HeroSearchProducts/SearchProducts.tsx

```diff
@@ -0,0 +1,49 @@
+import React from 'react';
+import { arrayIndexingWithLength } from '../../utils/arrayIndexingWithLength';
+import ProductShowcaseGrid from '../ProductShowcaseGrid';
+import Grid from '@material-ui/core/Grid';
+import { useStyles } from './SearchProducts.styles';
+import { AppBar, CssBaseline, Drawer } from '@material-ui/core';
+
+import SideBar from '../SideBar';
+import SearchBar from '../SearchBar';
+import NavBar from '../NavBar';
+const SearchProducts: React.FC = () => {
+  const classes = useStyles();
+
+  return (
+    <>
+      <div className={classes.root}>
+        <CssBaseline />
+        <AppBar position="fixed" className={classes.appBar}>
+          <NavBar />
+        </AppBar>
+
+        <Drawer
+          className={classes.drawer}
+          variant="permanent"
+          classes={{
+            paper: classes.drawerPaper,
+          }}
+        >
+          <div className={classes.drawerContainer}>
+            <SideBar />
```

Comment:
I think the sidebar should be in a container with the products, see the wireframes, its not all the way to the left.

Also take a look at: Material-ui Box, Container, grid, gridList, Hidden to make the layouts more easily adaptive to responsiveness https://material-ui.com/components/box/

Reply:
There is a style property that puts it in the a container